File: java/CheckOutApp.java
import java.util.Scanner;
import java.util.ArrayList;
import java.util.Arrays;

public class CheckOutApp{
public static void main(String[] args){

Scanner input = new Scanner(System.in);

//ArrayList<String> names = new ArrayList<>();
ArrayList<ArrayList<String>> productList = new ArrayList<>();
//ArrayList<Integer> price = new ArrayList<>();

String message = """
WELCOME TO SEMICOLON STORE:
===========================

Press 1 to enter the purchase details.
Press 2 to display customers invoice.
press 3 to print receipt.
press 0 to close app.
""";

boolean hold = true;
while(hold){
System.out.println(message);
int userChoice = input.nextInt();
System.out.println();
switch(userChoice){
case 1:
	String output = """
Enter each product detials in the user's cart...
================================================""";
	System.out.println(output);
	System.out.println();
	
	String product = null;
	int pieces = 0;
	double prices = 0;
	String decision = "yes";

	System.out.println("What is the customer's name?");
	String customerName = input.next();

	do{
	System.out.println("What did the user buy?");
	//System.out.println("rice\nbeans\ngarri\ntomato");
	product = input.next();

	System.out.println("How many pieces?");
	pieces = input.nextInt();

	System.out.println("How much per unit?");
	prices = input.nextDouble();

	productList = CheckOutAppFunction.addItem(product, pieces, prices);
	CheckOutAppFunction.calculation(pieces, prices);

	System.out.println("Add more items?");
	decision = input.next();

	}while(decision.equals("yes"));

	System.out.println("What is your name?");
	String cashierName = input.next();

	System.out.println("How much discount will he/she get?");
	int discount = input.nextInt();

	ArrayList<String> names = CheckOutAppFunction.named(customerName, cashierName);


	System.out.println(names);
	System.out.println(productList);


	break;

case 2:
	ArrayList<ArrayList<String>> itemList = CheckOutAppFunction.addMore();
	System.out.println(itemList);
	double [] sumed = CheckOutAppFunction.addMoreTotal();
	System.out.println(Arrays.toString(sumed));
	break;

case 0:
	hold = false;
	System.out.print("GoodBye");
	break;
default:
	System.out.print("Invalid input");


}


}
}

}

File: oldSnacks/python/CheckOutApp.py
import CheckOutAppFunction
import math

product_list = []
sumed = []
vated = 17.50;
billSum = 0;
payment = 0;
balance = 0;

message = """
WELCOME TO SEMICOLON STORE:
===========================

press 1 to enter the purchase details.
press 2 to display customers invoice.
press 3 to print receipt.
press 0 to close app.
""";

while True:
	print(message);
	userChoice = input();
	print();
	match(userChoice):
		case '1':
			print("Enter each product detials in the user's cart...\n================================================")
			print()
			product = '';
			pieces = 0;
			prices = 0;
			decision = "yes";

			print("What is the customer's name?");
			customerName = input();
			
			decision = 'yes';
			while True:
				print("What did the user buy?");
				product = input();

				print("How many pieces?");
				pieces = int(input());

				print("How much per unit?");
				prices = float(input());

				product_list = CheckOutAppFunction.addItem(product, pieces, prices);
				CheckOutAppFunction.calculation(pieces, prices);

				print("Add more items?");
				decision = input();

				if decision == 'no':
					break

			print("What is your name?");
			cashierName = input();

			print("How much discount will he/she get?");
			discount = int(input());
			discountSummed = CheckOutAppFunction.discountCalculation(discount);

			CheckOutAppFunction.named(customerName, cashierName);

		case '2':
			itemList = CheckOutAppFunction.addMore();
			print(f"\t {itemList} ");
			print();
			
			sumed = CheckOutAppFunction.addMoreTotal();
			num = sum(sumed)
			print(f"\t\t\t\t\tSub Total: {num}");

			#discountSummed = CheckOutAppFunction.discounts();
			print(f"\t\t\t\t\tDiscount: {discountSummed}");
			vated = (vated / 100) * num;
			print(f"\t\t\t\t\tVAT @ 17.50%: {vated}");
			print("=================================================================================================");
			billSum = billSum + num;
			billSum = billSum - discountSummed + vated;
			print(f"\t\t\t\t\tBill Total: {billSum}");
			print("=================================================================================================");
			print(f"THIS IS NOT AN RECEIPT KINDLY PAY {billSum}");
			print("=================================================================================================");
			print();
			print();
			print();
			print("How much did the customer give you?");
			payment = int(input());


		case '3':
			itemList = CheckOutAppFunction.addMore();
			print(f"\t {itemList} ");
			print();

			sumed = CheckOutAppFunction.addMoreTotal();
			num = sum(sumed)
			print(f"\t\t\t\t\tSub Total: {num}");
	
			#discountSummed = CheckOutAppFunction.discounts();
			print(f"\t\t\t\t\tDiscount: {discountSummed}");
			print(f"\t\t\t\t\tVAT @ 17.50%: {vated}");
			print("=================================================================================================");
			print(f"\t\t\t\t\tBill Total: {billSum}");

			print(f"\t\t\t\t\tAmount Paid {payment}");
			balance = billSum - payment;
			balance = abs(balance);
			print(f"\t\t\t\t\tBalance: {balance}");
			print("=================================================================================================");
			print("THANKS YOU FOR YOUR PATRONAGE");
			print("=================================================================================================");
			print();
			print();
			print();


		case '0':
			False;
			print("GoodBye");
			break
		case '_':
			print("Invalid input");

